saleor/graphql/schema.graphql

```diff
@@ -3107,6 +3092,21 @@ type PluginCountableEdge {
   cursor: String!
 }
 
+type PluginError {
```

Comment:
Not sure if we want to delete objects from the schema.  I assume that we should deprecated fields first. 
@maarcingebala  What do you think?

Reply:
It's fine, it's part of the dashboard API which we control and can fix. Unless you use it in integrations @korycins?